.gitignore

```diff
@@ -12,6 +12,7 @@
 .pub/
 build/
 **/pubspec.lock
+**/google-services.json
```

Comment:
Added this so that our google-services file doesnt get added to the github as it contains secrets file